tests/acceptance/pageObjects/sharedWithMePage.js

```diff
@@ -82,6 +82,14 @@ module.exports = {
         locateStrategy: this.elements.getSharedFromUserName.locateStrategy,
         selector: requiredXpath
       })
+    },
+    assertShareIsAbsent: function (element, sharer) {
```

Comment:
We decided to not have assert on page. Considering this PR to be small, I guess, you could just make one function that returns all shares, and assert on the context.